packages/backend/unique-ids-generator/src/generate-unique-id.ts

```diff
@@ -0,0 +1,32 @@
+import { createDebug } from "@gravitywelluk/debug";
+
+const debug = createDebug("UNIQUE_IDS_GENERATOR:GENERATE-UNIQUE-ID");
+
+interface GenerateUniqIdParams {
```

Comment:
Should this be `GenerateUniqueIdParams`?

Reply:
Good point !